docs/workflows/developing-workflow-tasks.md

```diff
@@ -28,10 +28,21 @@ ECS activities require a docker image. The docker image is defined as part of th
     publicIp: true
     docker:
       username: cumulususer
+      repoHost: dockerhub
     services:
       EcsTaskHelloWorld:
         image: cumuluss/cumulus-ecs-task:1.2.3
         cpu: 800
         memory: 1500
         count: 1
 ```
+
+### Specifying Repository Host (ECR | Dockerhub)
+
+Cumulus currently supports two methods of pulling images from a hosted repository by setting the `ecs.docker.repoHost` attribute to either `ecr` or `dockerhub`.
+
+*ecr* will use the IAM role attached to the instance (defined in `packages/deployment/iam/cloudformation.template.yml` under `ECSRole`) to authenticate against a repository hosted in AWS ECR. Using this method will still allow instances to pull images from *public* dockerhub repositories *without authentication.*
```

Comment:
as a nitpick, can we put a link to amazon's ECR documentation [AWS ECR][whatever the link is]?

Reply:
I think that's a good idea. Will do.